java/fury-format/src/test/java/org/apache/fury/format/type/DataTypeTest.java

```diff
@@ -27,7 +27,7 @@
 import org.apache.fury.test.bean.BeanA;
 import org.testng.annotations.Test;
 
-public class DataTypesTest {
+public class DataTypeTest {
```

Comment:
this is test suite for org.apache.fury.format.type.DataTypes, maybe we should keep name consist with `DataTypes`